.github/workflows/ci-pull-request.yml

```
@@ -8,21 +8,22 @@ env:
+        uses: ./.github/actions/node-build
+        with:
+          node_version: '18'
 
-    - name: Create Docker image
-      uses: ./.github/actions/docker-build-push
-      with:
-        push_to_ghcr: false
-        image_name: oriso-frontend
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker build validation without publishing
+        uses: ./.github/actions/docker-build-push
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/docker-build-push": line 4: unexpected key "type" for definition of input "dockerfile"

(action)